package.json

```diff
@@ -6,8 +6,8 @@
   "module": "index.ts",
   "scripts": {
     "build": "skyux build-public-library",
-    "test": "skyux test",
-    "test:watch": "skyux watch",
+    "test": "skyux test --coverage library",
+    "test:watch": "skyux watch --coverage library",
```

Comment:
I realize this different, but do you have any opinion on using `watch` instead `test:watch`?

Reply:
Done.